src/platform/datapath_raw_xdp_win.c

```diff
@@ -943,12 +948,6 @@ CxPlatDpRawInitialize(
     }
 
     CxPlatListInitializeHead(&Xdp->Interfaces);
-    if (QUIC_FAILED(XdpLoadApi(XDP_API_VERSION_1, &Xdp->XdpApiLoadContext, &Xdp->XdpApi))) {
```

Comment:
A couple thoughts:

1. This used to be the first place of failure if XDP wasn't installed. Now it's going to be down in `CxPlatDpRawInterfaceInitialize`, right? What is the error that's returned in this case?
2. We are firing off the `XdpInitialize` event below before we ever even try to call into XDP. I'm thinking we should move this down now, likely to the end of the function, and only if we can successfully call into XDP.

Reply:
1. XdpInterfaceOpen returns 0x80070002. WinDbg says this is "The system cannot find the file specified."
2. Which `XdpInitialize` are you talking about? there is not such call.

Reply:
1. I am thinking if we get `0x80070002`, which would be `HRESULT_FROM_WIN32(ERROR_FILE_NOT_FOUND)`, for the very first interface, we should treat that as `QUIC_STATUS_NOT_SUPPORTED`.

2. I am referring to:
```c
    QuicTraceLogVerbose(
        XdpInitialize,
        "[ xdp][%p] XDP initialized, %u procs",
        Xdp,
        Xdp->PartitionCount);
```